src/Microsoft.IdentityModel.Clients.ActiveDirectory/Internal/Http/WebProxyProvider.cs

```diff
@@ -29,12 +29,24 @@
 
 namespace Microsoft.IdentityModel.Clients.ActiveDirectory
 {
+#if ANDROID
     [Android.Runtime.Preserve(AllMembers = true)]
-    internal class WebProxyProvider : IWebProxyProvider
+#endif
+    internal class WebProxyProvider
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a 'protected' constructor or the 'static' keyword to the class declaration. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://iddp-sonarqube:9000/coding_rules#rule_key=csharpsquid%3AS1118)